preact/example-customer-account--wishlist--preact/extensions/wishlist-profile-block/src/ProfileBlockExtension.jsx

```diff
@@ -0,0 +1,21 @@
+import '@shopify/ui-extensions/preact';
+import {render} from 'preact';
+
+// [START block.build-ui]
+export default async () => {
+  render(<BlockExtension />, document.body);
+};
+
+function BlockExtension() {
+  return (
+    <s-section>
+      <s-stack direction="inline" inline-alignment="center" gap="small-400">
+        <s-text>Grow your garden with more plants from your wishlist.</s-text>
+        <s-link href="extension:wishlist-extension-preact/">
```

Comment:
Is `wishlist-extension-preact` an extension we'll actually have?

Reply:
maybe that's a stupid question then, but why isn't that other extension part of this PR?

Reply:
Im just splitting them up to make them easier to review. The other one is a full page extension that will need a bit more careful reviewing